includes/admin/class-wc-rest-woopay-session-controller.php

```diff
@@ -21,14 +21,14 @@ class WC_REST_WooPay_Session_Controller extends WP_REST_Controller {
 	 *
 	 * @var string
 	 */
-	protected $namespace = 'wc/v3';
+	protected $namespace = 'wc/woopay';
```

Comment:
Isn't it better change `wc` to `wc-pay` or something else so the namespace has nothing to do with Woo Core?

Reply:
Updated to `payments` on https://github.com/Automattic/woocommerce-payments/pull/9317/commits/7a73a700272542ac5beb9fa14d7e20482b58ac51 to keep consistency with other WCPay routes.